image/nodejs.sh

```diff
@@ -4,8 +4,13 @@ source /pd_build/buildconfig
 set -x
 
 echo "+ Enabling Node Source APT repo"
-curl -sL https://deb.nodesource.com/setup_18.x | bash -
```

Comment:
no longer using the script, but still fetching from the same source. just more manually... hmm